### PR TITLE
Don't filter EV alignment

### PR DESCRIPTION
**Pull request by nicovanduijn**
This removes the IIR filter on the attitude offset between vision and EKF frame.
I don't see why this rotation would need to be low pass filtered. Especially if one of the two drifts, this leads to toilet-bowling.

Not sure if I did this right, so any review is appreciated.

### CODE REVIEW

File: EKF/ekf_helper.cpp [hunk 1]
```diff
@@ -1615,32 +1615,6 @@ void Ekf::calcExtVisRotMat()
...
-
-		if (input_delta_len > 0.1f) {
-			rot_vec = _ev_rot_vec_filt + _input_delta_vec * (0.1f / input_delta_len);
-		}
-
-		// Apply a first order IIR low pass filter
-		const float omega_lpf_us = 0.2e-6f; // cutoff frequency in rad/uSec
-		float alpha = math::constrain(omega_lpf_us * (float)(_time_last_imu - _ev_rot_last_time_us), 0.0f, 1.0f);
-		_ev_rot_last_time_us = _time_last_imu;
-		_ev_rot_vec_filt = _ev_rot_vec_filt * (1.0f - alpha) + rot_vec * alpha;
```
Comment (kamilritz) [inline EKF/ekf_helper.cpp]: _ev_rot_vec_filt should then be removed.

**Comment by nicovanduijn** [comment]
Yeah, e.g. the log that triggered this change in the first place:
https://review.px4.io/plot_app?log=21099c0f-75f8-48b0-af7e-0071e0d50089

**Comment by LorenzMeier** [comment]
I think you need to rebase and force-push.

**Comment by bresch** [comment]
**Note for everyone**: I didn't merge that to ECL master but to a testing branch. We will open a PR to master if the tests show good results.

**Pull request merged**